### ISSUE TITLE
respawn mobs no longer attack/link/move once attacked

### ISSUE THREAD

**Issue by lichtl** [DarkstarProject/darkstar#2558]
It seems that if a mob respawn it no longer agro/attack/link/move. They do roam however once you attack them they do not response at all.

![ffxi_2016 01 04_23 35 53](https://cloud.githubusercontent.com/assets/1228155/12102606/457c4d9c-b33c-11e5-89f6-97be2ecbd232.jpg)
![ffxi_2016 01 04_23 38 30](https://cloud.githubusercontent.com/assets/1228155/12102607/459830d4-b33c-11e5-896b-e786ee594fee.jpg)

**Comment by teschnei** [comment]
Can you tell me if you are still seeing this as of latest? I did notice it happening to myself before it was reported, but I can't seem to reproduce it anymore (may have accidentally fixed it)

**Comment by lichtl** [comment]
Just updated to the latest and its still there.

[Info] DarkStar - Git Revision Hash: 28f6c525fb7bcff7c312799a9605d5f6a315ddb9.
@darkstar:~$ uname -a
Linux darkstar 3.16.0-57-generic #77~14.04.1-Ubuntu SMP Thu Dec 17 23:20:00 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
@darkstar:~$ g++ --version
g++ (Ubuntu 5.3.0-3ubuntu1~14.04) 5.3.0 20151204
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

**Comment by teschnei** [comment]
do you know of a quick way to get it to happen by any chance? I could only
see it happen randomly, and that's not good for testing

**Comment by ymkc** [comment]
Got the same issue on the same revision, easily reproduceable in palborogh mines with the quadavs.

As soon as they repop, you cannot get them to engage/aggro/link. Tried attacking/casting/resting

**Comment by lichtl** [comment]
@teschnei I'm able to reproduce it at any zone. Tried it again earlier in Aery with the Darters

**Comment by teschnei** [comment]
sorry, I should have been more specific: are you killing them and letting
them respawn naturally, or despawning/respawning them with a command, or
does it happen on their initial spawn on server boot?

**Comment by lichtl** [comment]
I killed the inital ones and let them respan naturally. Even after killing the ones which don't act the next respan also don't perform any actions once attacked (however they roam after repop)

**Comment by lichtl** [comment]
just performed some more testing. In addition I now confirm that timed NM dont seem to spawn any longer (Either no Kings,Serket,Simurth, being up after around 11h server uptime.

I can confirm that this only appear once they have been killed. I've zoned 2 (one being attacked while the other being linked) the aggroed one despawned after zoneing and took 15 minutes to repop (I think this is also wrong as I recall they should repop within the next minute) but acted in a normal way (aggro'd, linked and attacked). I've confirmed them by using @getid .

The attacked one I'm currently wait for repop (20 minutes since then and no reappearing). I will update the post once it repoped
